lib/mini_profiler/config.rb

```diff
@@ -37,6 +37,8 @@ def self.default
           @max_sql_param_length = 0 # disable sql parameter collection by default
           @skip_sql_param_names = /password/ # skips parameters with the name password by default
           @enable_advanced_debugging_tools = false
+          @snapshot_every_n_requests = -1
+          @snapshots_limit = 1000
```

Comment:
Is this a good/safe default value @SamSaffron? Note this feature is off by default, but when it's turned on this config specifies how many snapshots Mini Profiler is allowed to keep.

Reply:
Yes I think this is a good default!